Mathlib/Data/Finset/Sort.lean

```diff
@@ -69,6 +69,56 @@ theorem sort_perm_toList (s : Finset α) : sort r s ~ s.toList := by
   rw [← Multiset.coe_eq_coe]
   simp only [coe_toList, sort_eq]
 
+theorem filter_sort_commute [DecidableEq α] (f : α → Prop) [DecidablePred f] (s : Finset α) :
+    sort r (filter (f ·) s) = List.filter f (sort r s) := by
```

Comment:
It's not really a commutativity lemma as it talks about two different `filter`s here
```suggestion
theorem sort_filter [DecidableEq α] (f : α → Prop) [DecidablePred f] (s : Finset α) :
    sort r (filter (f ·) s) = List.filter f (sort r s) := by
```